semmc-ppc/src/SemMC/Architecture/PPC32.hs

```diff
@@ -336,23 +337,23 @@ operandValue :: forall sym s.
 operandValue sym locLookup op = TaggedExpr <$> operandValue' op
   where operandValue' :: PPC.Operand s -> IO (A.AllocatedOperand PPC sym s)
         operandValue' (PPC.Abscalltarget (PPC.ABT absTarget)) =
-          A.ValueOperand <$> S.bvLit sym knownNat (toInteger absTarget)
+          A.ValueOperand <$> S.bvLit sym knownNat (BV.trunc knownNat (BV.word32 absTarget))
```

Comment:
Could we get a comment around here about why the use of trunc?  Is it the case that the dismantle encoding of branch targets is imprecise and Word32?

Reply:
Good point -- the idea here is to keep as much of the truncation in `bv-sized` as possible. We could have first converted `absTarget` from `Word32` to `Integer`, then simply called `BV.mkBV`, but this makes it explicit that `absTarget` is losslessly converted to a bitvector, than truncated to whatever width `knownNat` is.

I'll put a comment at the top of the function conveying that much.